src/Statement/StatementGuid.php

```diff
@@ -46,6 +51,7 @@ public function __construct( EntityId $entityId, $guid ) {
 
 		$this->serialization = $entityId->getSerialization() . self::SEPARATOR . $guid;
 		$this->entityId = $entityId;
+		$this->guid = $guid;
```

Comment:
Lets call this guidPart just to avoid any confusion for people looking in there for the first time.